CrazyCanvas/Source/ECS/Systems/GUI/HUDSystem.cpp

```diff
@@ -1,3 +1,4 @@
+#include "Application/API/CommonApplication.h"
```

Comment:
Is this used in HUDSystem. Either my diff is wrong on Github, or this is the only change in this file?